src/easier68k/core/opcodes/dc.py

```diff
@@ -0,0 +1,392 @@
+                        hexed = '0' + hexed
+
+                if size == 'W':
+                    while len(hexed) % 4 != 0:
+                        hexed = '0' + hexed
+
+                if size == 'B':
+                    while len(hexed) % 2 != 0:
+                        hexed = '0' + hexed
+
+                # Length is now for sure even
+                for i in range(0, len(hexed), 2):
+                    params.append('$' + hexed[i:i+2])
+
+        return DC(params, size), issues
+
+    @staticmethod
+    def is_valid(command: str, parameters: str) -> (bool, list):
+        """
+        Tests whether the given command is valid
+
+        >>> DC.is_valid('DC.B', '\\'Hello, world!\\'')[0]
+        True
+
+        >>> DC.is_valid('DC.B', '\\'Hello\\'\\' world!\\'')[0]
+        False
+
+        >>> DC.is_valid('DC.W', '\\'Hello\\', \\' world!\\'')[0]
+        True
+
+        >>> DC.is_valid('DC.W', '$0A, $0B')[0]
+        True
+
+        >>> DC.is_valid('DC.L', '$0A')[0]
+        True
+
+        >>> DC.is_valid('DC.B', '')[0]
+        False
+
+        >>> DC.is_valid('DC.W', '\\'Hey!\\', $0A')[0]
+        True
+
+        >>> DC.is_valid('DC.G', '$0A')[0]
+        False
+
+        :param command: The command itself (e.g. 'MOVE.B', 'LEA', etc.)
+        :param parameters: The parameters after the command (such as the source and destination of a move)
+        :return: Whether the given command is valid and a list of issues/warnings encountered
+        """
+        issues = []
+        try:
+            assert opcode_util.check_valid_command(command, 'DC', valid_sizes=DC.allowed_sizes), 'Command invalid'
+            # Can't just do a "split by comma" here because we could have commas inside of a string literal
+            param_count = 0
+            quote_delim = None  # If None, we're not in a quote: if it has a value, we know what we're looking for to break the quote
+            current_param = ''
+            is_string = False
+
+            for c in parameters:
+                if c == ',' and not quote_delim:  # End of this parameter (and not a comma in a quote)
+                    if current_param:
+                        if not is_string:  # Try parsing this value for a literal (if it's a string it's almost certainly fine)
+                            assert parse_literal(current_param.strip()) is not None, 'Error parsing literal'
+                        param_count += 1
+
+                    is_string = False
+                    quote_delim = None
+                    current_param = ''
+                    continue
+
+                if quote_delim and c == quote_delim:  # Breaking out of the quote
+                    quote_delim = None
+                    continue
+
+                if c == '\'':
+                    if not quote_delim:  # This is the start of a quote
+                        quote_delim = '\''
+                        assert not current_param.strip(), "Can't have two quotes in a row in one parameter"
```

Comment:
I would re-word this assert, I'm unsure what it is trying to tell me. Maybe instead say "Cannot have an empty string?"

Reply:
It's mainly trying to say that I can't do, for example `'Hello' ' world'`, defining two string literals in a row without them being separated by a comma. I might change it to "Can't have two string literals in a row without being separated by a comma", which while still kinda weird (I can't think of a good way to put this error) should do a bit better.

Reply:
Why not something like "expected comma between two string literals" ?

Reply:
..... I like that much better, that was obvious, let's go with that